search_action_domain/models/ir_actions_server.py

```diff
@@ -0,0 +1,25 @@
+from odoo import models
+
+
+class IrActionsServer(models.Model):
+    _inherit = "ir.actions.server"
+
+    def create_as_ir_filter(self):
+        for record in self.filtered(lambda r: r.state == "code"):
+            self.env["ir.filters"]._load_records(
```

Comment:
This can receive a list, you can pass all at once.

Maybe a list comprehension

```
self.env["ir.filters"]._load_records(
    [
        ...
    ]
    for record in self.filtered(lambda r: r.state == "code")
)
```